File: appthemehelper/src/main/java/code/name/monkey/appthemehelper/common/ATHActionBarActivity.java
package code.name.monkey.appthemehelper.common;

import androidx.appcompat.widget.Toolbar;

import code.name.monkey.appthemehelper.util.ToolbarContentTintHelper;

public class ATHActionBarActivity extends ATHToolbarActivity {

    @Override
    protected Toolbar getATHToolbar() {
        return ToolbarContentTintHelper.getSupportActionBarView(getSupportActionBar());
    }
}

File: appthemehelper/src/main/java/code/name/monkey/appthemehelper/common/ATHActionBarActivity.kt
package code.name.monkey.appthemehelper.common

import androidx.appcompat.widget.Toolbar

import code.name.monkey.appthemehelper.util.ToolbarContentTintHelper

class ATHActionBarActivity : ATHToolbarActivity() {

    override fun getATHToolbar(): Toolbar? {
        return ToolbarContentTintHelper.getSupportActionBarView(supportActionBar)
    }
}
